### ISSUE TITLE
[a11y] Cannot open workbench data sets

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1345]
**Describe the bug**
When I attempt to open the WB data sets, I get the following error:
![image](https://user-images.githubusercontent.com/37256050/165797110-5b30e02e-8b69-4ed6-a9d6-41338d3975b3.png)

[Error Message](https://drive.google.com/open?id=1QoJARPKFA3Mi3NZhEMRUMT6B9U1C1lWo&authuser=grant%40specifysoftware.org&usp=drive_fs)

**To Reproduce**
1. Go to `newmexico`
2. Click on **WorkBench**
3. See error

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge

**Comment by maxxxxxdlp** [comment]
We figured out that this was caused by database being corrupted though running incompatible migrations from the uploader_memory_optimizations branch
